stackslib/src/net/api/posttransaction.rs

```diff
@@ -162,7 +162,7 @@ impl HttpRequest for RPCPostTransactionRequestHandler {
             }
             _ => {
                 return Err(Error::DecodeError(
-                    "Wrong Content-Type for transaction; expected application/json".to_string(),
+                    "Wrong Content-Type for transaction; expected application/json or application/octet-stream".to_string(),
```

Comment:
You removed the `application/octet-stream` option

Reply:
That's not true for `posttransaction`, though -- I think that still accepts an `octet-stream`

Reply:
Oh yeah I didn't check the filename there... that's a message I had fixed for a *different* endpoint